Makefile

```diff
@@ -82,14 +93,15 @@ AS_INCLUDES :=
 C_INCLUDES  := $(addprefix -I,                            \
 	$(sort $(dir $(C_HEADERS)))                           \
 	$(sort $(dir $(shell find $(CUBE_DIR) -name "*.h")))  \
-)                                                         \
+	$(sort $(dir $(TEST_HEADERS)))                        \
```

Comment:
Só deveria adicionar isso se for teste, a aplicação normal não devia ter acesso as coisas do teste
No caso não seria só adicionar caso TEST=1, pq ai quando tivesse compilando pra testes as coisas normais ainda teriam o acesso, cria uma variável separada só pra flag de teste e coloca ela só na compilação dos arquivos de teste